frontend-ssr/components/cli_git/match_commands.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+
+// class MatchWords extends React.Component {
+//     constructor(props) {
+//         super(props);
+
+//         this.state = {
+//             message: '',
+//         }
+//     }
+
+//     handleSubmit = (event) => {
+//         event.preventDefault();
+
+//         const inputValue = document.querySelector('[name="input_text"]').value;
+//         const {defaultValue} = this.props;
+
+//         if (Array.isArray(defaultValue)) {
+//             var result = defaultValue.some(
+//                 currentValue => currentValue.toLowerCase() === inputValue.trim().toLowerCase()
+//             );
+//         }
+
+//         if (typeof defaultValue === 'string') {
+//            var wordToMatch = inputValue.trim().toLowerCase() === defaultValue.toLowerCase();
+//         }
+
+//         if (result || wordToMatch) {
+//             this.setState({
+//                 message: 'you guess it'
+//             });
+//         } else {
+//             this.setState({
+//                 message: 'try again'
+//             });
+//         }
+//     }
+
+
+//     render() {
+//         const { message } = this.state;
+//         return (
+//             <form onSubmit={this.handleSubmit}>
+//                 <input
+//                     type="text"
+//                     name="input_text"
```

Comment:
The name is too generic. Usually we want it to reflect what's in the input. For example `password` for password inputs, `email` for emails, etc. Here, let's call it something related to Git, since you're typing GIT commands. Maybe `git_command`, `cli_command` etc.